CHANGELOG-3.4.md

```diff
@@ -438,6 +440,8 @@ Note: **v3.5 will deprecate `etcd --log-package-levels` flag for `capnslog`**; `
   - Especially, gRPC proxy was affected, since it detects a leader loss with a key `"proxy-namespace__lostleader"` and a watch revision `"int64(math.MaxInt64 - 2)"`.
   - Now, this server-side panic has been fixed.
 - Fix [memory leak in cache layer](https://github.com/etcd-io/etcd/pull/10327).
+- Change [gRPC proxy to expose endpoint /metrics](https://github.com/etcd-io/etcd/pull/10618).
```

Comment:
Can we be more clear? `expose etcd server endpoint`

Reply:
fixed